packages/web3-providers-http/src/index.ts

```diff
@@ -138,16 +138,32 @@ export default class Web3ProvidersHttp
         return false;
     }
 
-    /**
-     * Makes an Axios POST request using provided {args}
-     *
-     * @param args RPC options, request params, AxiosConfig
-     * @returns
-     */
-    async request(args: RequestArguments): Promise<RpcResponse> {
+    private async _eth2Request(args: RequestArguments): Promise<AxiosResponse> {
```

Comment:
function documentation comments required, 
plus as suggested earlier you should define your own type instead of `RequestArguments` for better aligning with eth2 requests ( removing `method` and adding `endpoint` ..etc )